docs/DeveloperGuide.md

```diff
@@ -257,329 +257,381 @@ _{Explain here how the data archiving feature will be implemented}_
+| S6  | `HIGH`   | user                                       | add remarks to a student                   | keep track of supplementary information                                |
+| S7  | `MEDIUM` | user                                       | keep track of a student's school           |                                                                        |
+| S8  | `MEDIUM` | user                                       | keep track of a student's academic level   | be reminded of the content that is relevant to the student             |
+| S9  | `MEDIUM` | user                                       | keep track of a student's outstanding fees | be reminded of the fees I have yet to collect                          |
+| S10 | `LOW`    | user                                       | have an archive for removed students       | revisit the archived data if the need arises                           |
+| S11 | `LOW`    | user with many students stored in **TAB**  | sort students by name                      | locate a student easily                                                |
+| S12 | `LOW`    | user with many students stored in **TAB**  | sort students by tag                       | locate a student easily                                                |
+| L1  | `HIGH`   | user                                       | schedule recurring lessons for a student   | keep track of when the student's lessons are                           |
+| L2  | `HIGH`   | user                                       | delete a student's lesson                  | remove lessons that I no longer have                                   |
+| L3  | `MEDIUM` | user                                       | edit a student's lesson                    | update lesson information                                              |
+| L4  | `MEDIUM` | user                                       | add makeup lessons for a student           | differentiate between the two different lessons                        |
+| L5  | `MEDIUM` | user                                       | view the lessons I have for the week       | easily manage my schedule                                              |
+| L6  | `MEDIUM` | user                                       | view a calendar of my scheduled lessons    | plan ahead while managing my schedule                                  |
+| U1  | `HIGH`   | new user                                   | see usage instructions                     | refer to instructions whenever I need guidance on how to use **TAB**   |
+| U2  | `HIGH`   | new user                                   | purge all current data                     | remove all sample/experimental data I used while exploring **TAB**     |
+| U3  | `HIGH`   | user                                       | find a student by name                     | locate details of persons without having to go through the entire list |
```

Comment:
There should be another row for find a student by student details?

Reply:
I'll change the user story for finding by name to finding by keyword, and the corresponding UC